packages/agoric-cli/src/publish.js

```diff
@@ -1,6 +1,48 @@
 // @ts-check
 /// <reference types="ses"/>
 
+import { E } from '@endo/far';
+
+import {
+  iterateEach,
+  makeFollower,
+  makeLeaderFromRpcAddresses,
+  makeCastingSpec,
+} from '@agoric/casting';
+import { DirectSecp256k1HdWallet, Registry } from '@cosmjs/proto-signing';
+import { defaultRegistryTypes } from '@cosmjs/stargate';
+import { stringToPath } from '@cosmjs/crypto';
+import { Decimal } from '@cosmjs/math';
+import { Bech32 } from '@cosmjs/encoding';
+
+import { MsgInstallBundle } from '@agoric/cosmic-proto/swingset/msgs.js';
+
+// https://github.com/Agoric/agoric-sdk/blob/master/golang/cosmos/daemon/main.go
+const Agoric = {
+  Bech32MainPrefix: 'agoric',
+  CoinType: 564,
+  proto: {
+    swingset: {
+      InstallBundle: {
+        // matches package agoric.swingset in swingset/msgs.go
+        typeUrl: '/agoric.swingset.MsgInstallBundle',
+      },
+    },
+  },
+  // arbitrary fee for installing a bundle
+  fee: { amount: [], gas: '50000000' },
```

Comment:
This shouldn't be hard-coded; we do have near to medium term plans to charge by the byte: #6097

Reply:
not critical; seems reasonable for it to come in a follow-on PR, once requirements become more clear